.pre-commit-config.yaml

```diff
@@ -187,15 +187,15 @@ repos:
 
   - repo: local
     hooks:
-
       - id: docs-plugin-modules
         name: Build documentation for collection modules and action plugins.
         entry: >-
           ansible-doc-extractor --template docs/templates/plugin-docs.j2
           --markdown "docs/plugins/Modules_and_action_plugins/"
         language: python
         types: [python]
-        additional_dependencies: ['ansible-doc-extractor>=0.1.10', 'ansible-core>=2.16.0,<2.19.0']
+        additional_dependencies:
+          ["ansible-doc-extractor>=0.1.10", "ansible-core>=2.16.0,<2.19.0"]
```

Comment:
please disable your linter to prevent these changes, or make it convert it properly to a yaml list. This block style with a flow list makes zero sense.